aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -332,14 +333,25 @@ module std::features {
         is_enabled(BN254_STRUCTURES)
     }
 
+    /// If enabled:
```

Comment:
Same, should we check the feature flag is enabled when calling these new functions?

Reply:
they are designed to work in both cases so we don't have to.